tests/test_series.py

```diff
@@ -1870,7 +1870,7 @@ def test_change_to_dict_return_type() -> None:
     ("longfloat", np.longdouble),
     ("g", np.longdouble),
     ("f16", np.longdouble),
-    ("float96", np.longdouble),  # NOTE: WINDOWS ONLY
+    # ("float96", np.longdouble),  # NOTE: WINDOWS ONLY
```

Comment:
Maybe delete this line - or at least change the comment to indicate why you commented it out?

Reply:
It's changed to NOTE: unsupported in the last commit

Reply:
oh I see you did this in a later commit